### PR TITLE
MODLD-332: Convert marc field 008, char 18-21 to "illustrations" edge…

### PR DESCRIPTION
**Pull request by askhat-abishev**
… of Work

### CODE REVIEW

**Review by pkjacob — APPROVED** [review]

File: src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java [hunk 1]
```diff
@@ -666,9 +667,30 @@ private void validateWork(ResourceEdge edge) {
...
 
+  private void validateBook(ResourceEdge edge, PredicateDictionary predicate,
+                            Map<String, List<String>> categoryProperties,
+                            Map<String, List<String>> categorySetProperties) {
+    validateEdge(edge, predicate, List.of(CATEGORY), categoryProperties,
+      categoryProperties.get(TERM.getValue()).get(0));
+    var outgoingIterator = edge.getTarget().getOutgoingEdges().iterator();
+    validateEdge(outgoingIterator.next(), IS_DEFINED_BY, List.of(CATEGORY_SET), categorySetProperties,
+      categorySetProperties.get(LABEL.getValue()).get(0));
+    assertThat(outgoingIterator.hasNext()).isFalse();
```
Comment (AndreiBordak) [inline src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java:691]: can it be like:
```
assertThat(edge.getTarget().getOutgoingEdges())
      .hasSize(1)
      .singleElement()
      .satisfies(resEdge -> 
        validateEdge(resEdge, IS_DEFINED_BY, List.of(CATEGORY_SET), categorySetProperties, 
          categorySetProperties.get(LABEL.getValue()).get(0)));
```

File: src/test/java/org/folio/marc4ld/mapper/field008/Marc2LdIllustrationsIT.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        validateEdge(edges.get(1), ILLUSTRATIONS, List.of(CATEGORY),
+          Map.of(
+          "http://bibfra.me/vocab/marc/code", List.of("b"),
+          "http://bibfra.me/vocab/lite/link", List.of("http://id.loc.gov/vocabulary/millus/map"),
+          "http://bibfra.me/vocab/marc/term", List.of("Maps")
+          ), "Maps");
+      })
+      .extracting(edges -> getOutgoingEdges(edges.get(0)))
+      .satisfies(edges -> {
+        assertEquals(1, edges.size());
```
Comment (AndreiBordak) [inline src/test/java/org/folio/marc4ld/mapper/field008/Marc2LdIllustrationsIT.java:51]: can be replaced with hasSize(1) after line#49

Comment (askhat-abishev) [reply src/test/java/org/folio/marc4ld/mapper/field008/Marc2LdIllustrationsIT.java]: done

File: src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java (hunk 1 above)
Comment (askhat-abishev) [reply src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java]: yes, it can
thank you, changed

**Pull request merged**